src/strategy.rs

```diff
@@ -107,7 +107,7 @@ pub fn partial_rollout(group: &str, variable: Option<&String>, rollout: u32) ->
         100 => true,
         rollout => {
             if let Ok(normalised) = normalised_hash(group, variable, 100) {
-                rollout > normalised
+                rollout >= normalised
```

Comment:
This appears to have been a bug in the algorithm and was also present for the variant distribution.